src/stylesheets/elements/form-controls/_checkbox-and-radio.scss

```diff
@@ -208,6 +231,14 @@
         content: "\2714";
         text-align: center;
       }
+
+      @media (forced-colors: active) {
+        background-color: ButtonText;
+
+        &--tile {
+          background-color: ButtonText;
+        }
+      }
```

Comment:
# Checkbox and Radio Buttons

Currently, the radio button and checkbox components are missing important visual queues for users implementing Windows High Contrast mode including the buttons/boxes themselves, when a selection has been made, and a visual queue of disabled form options.

Because most of the stylings are shared between Radio + Checkbox, it was easiest to fix together.

## Radio Buttons

**Currently:**

Default

![image](https://user-images.githubusercontent.com/25211650/153276569-dc69554d-6ad2-49c8-bd34-e1e7eee62252.png)

Tile

![image](https://user-images.githubusercontent.com/25211650/153276968-9e76c44a-393c-4003-a3a5-bbfebd045603.png)

**After Fix:**

_// Note: Bolder border around Radio button / checkbox indicates focus_

Default:

![image](https://user-images.githubusercontent.com/25211650/153277508-be258731-7e99-4bfb-b95c-e5b98a1e3b85.png)

Tile:

![image](https://user-images.githubusercontent.com/25211650/153673621-bb2031bb-7b80-4b3a-b079-befdb7d34f5b.png)

### Checkbox

**Currently**

Default:

![image](https://user-images.githubusercontent.com/25211650/153277962-b47c6391-919b-4e47-b54f-a75e24c8bc47.png)

Tile:

![image](https://user-images.githubusercontent.com/25211650/153278101-19caf43e-8982-45b4-94e7-24f5ebd57b8b.png)

**After Fix**

Default:

![image](https://user-images.githubusercontent.com/25211650/153672482-352ff94d-b0e5-41ca-925b-4db5207360ae.png)


Tile:

![image](https://user-images.githubusercontent.com/25211650/153672514-4ece7215-5dd3-47d6-ae07-7a27f57de5cf.png)

The checkmark will not show in both cases due to the limitations of forced-colors mode being able to determine light or dark themes, to switch between the light and dark versions of the checkmark.

Our thinking was that the dark high contrast themes are more frequently used than the light themes, so the broader user base will get the most similar checkboxes, stylistically speaking, while the light users still maintain a functional componen